sdk/storage/storage-file-share/src/ShareFileClient.ts

```diff
@@ -1019,7 +1019,7 @@ export class ShareFileClient extends StorageClient {
    * Creates a new file or replaces a file. Note it only initializes the file with no content.
    * @see https://docs.microsoft.com/en-us/rest/api/storageservices/create-file
    *
-   * @param {number} size Specifies the maximum size in bytes for the file, up to 1 TB.
```

Comment:
There is only log optimization?

Reply:
Yes. As we already updated the constants when doing it last time for premium file. So this time, just added some tests.